orchestra/tests/unit/specs/mistral/test_workflow_spec_validate.py

```diff
@@ -84,34 +57,46 @@ def test_empty_task_list(self):
         self.assertDictEqual(wf_spec.inspect(), expected_errors)
 
     def test_task_default_list(self):
-        wf_def = """
-            version: '2.0'
-
-            sequential:
-                description: A basic sequential workflow.
-                task-defaults:
-                    timeout: 60
-                tasks:
-                    task1:
-                        action: std.noop
-        """
-
+        # load definition from fixture: tests/fixtures/workflows/mistral/xxx.yaml
+        wf_def = self.get_wf_def("task-defaults-valid")
         wf_spec = self.instantiate(wf_def)
 
         self.assertDictEqual(wf_spec.inspect(), {})
 
     def test_empty_task_default_list(self):
-        wf_def = """
-            version: '2.0'
+        # load definition from fixture: tests/fixtures/workflows/mistral/xxx.yaml
+        wf_def = self.get_wf_def("task-defaults-empty")
+        wf_spec = self.instantiate(wf_def)
+
+        self.assertDictEqual(wf_spec.inspect(), {})
 
-            sequential:
-                description: A basic sequential workflow.
-                task-defaults: {}
-                tasks:
-                    task1:
-                        action: std.noop
-        """
```

Comment:
We only put WF definition to `orchestra/tests/fixtures/workflows/mistral` if it will be commonly used.  If the WF definition is only used in only 1 test, we inline the definition in the test for easier references. Also, this change doesn't seem to have anything to do w/ this PR.